packages/account/src/BiconomySmartAccountV2.ts

```diff
@@ -84,6 +84,12 @@ export class BiconomySmartAccountV2 extends BaseSmartAccount {
     const instance = new BiconomySmartAccountV2(biconomySmartAccountConfig);
     instance.factoryAddress = biconomySmartAccountConfig.factoryAddress ?? DEFAULT_BICONOMY_FACTORY_ADDRESS; // This would be fetched from V2
 
+    if (biconomySmartAccountConfig.apiKey) {
```

Comment:
if biconomySmartAccountConfig has paymaster instance then assign it 
if it has apiKey then do this 

also let's rename apiKey to something like biconomyPaymasterApiKey